src/dotnet/UI.Blazor/Pages/Emails/EmailTemplatesTestPage.razor

```diff
@@ -8,29 +9,39 @@
 <Form Model="@_form">
     <FormBlock>
         <FormSection
-            For="() => _model.Selected"
+            For="() => _model.SelectedTemplate"
             InputId="template"
             IsLabelInsideInput="true"
             HideValidationMessage="true"
             Label="Template">
             <InputSelect
-                Value="_model.Selected"
-                ValueExpression="@(() => _model.Selected)">
+                Value="_model.SelectedTemplateName"
+                ValueExpression="@(() => _model.SelectedTemplateName)"
+                ValueChanged="@((string value) => OnSelectedTemplateChanged(value))">
                 @foreach (var template in _model.Templates) {
-                    <option value="@template">@template.Name</option>
+                    <option value="@template.Name">@template.Name</option>
                 }
             </InputSelect>
         </FormSection>
     </FormBlock>
 </Form>
 
-<MjmlHtmlPreview ComponentType="@(_model.Selected.ComponentType)"/>
+@if (_model.SelectedTemplate is null)
+    return;
+
+@if (_model.SelectedTemplate.Name == "Digest") {
+    <Button Click="SendDigest">Send Digest</Button>
+}
+
+<MjmlHtmlPreview ComponentType="@(_model.SelectedTemplate.ComponentType)"/>
 
 @code {
     private Model _model = null!;
     private FormModel _form = null!;
 
     [Inject] private ComponentIdGenerator ComponentIdGenerator { get; init; } = null!;
+    [Inject] private IAccounts Accounts { get; init; } = null!;
```

Comment:
Use IAccounts from Hub.Accounts instead of injection

Reply:
Done